docs/API.md

```diff
@@ -553,6 +554,50 @@ try {
 }
 ```
 
+<a name="listenBucketNotification"></a>
+### listenBucketNotification(String bucketName, String prefix, String suffix, String[] events, BucketEventListener listener)
+`public void listenBucketNotification(String bucketName, String prefix, String suffix, String[] events, BucketEventListener listener)`
```

Comment:
My choice here is like `public Iterable<Result<Event>> listenBucketNotification(String bucketName, String prefix, String suffix, EventType[] events)` .  Its just a thought.

Reply:
> My choice here is like `public Iterable<Result<Event>> listenBucketNotification(String bucketName, String prefix, String suffix, EventType[] events)` . Its just a thought.

One problem with Iterable is that it doesn't have close/free resources concept which means we cannot close the underlying resources (opened connection to the server)